lib/route/src/Obelisk/Route.hs

```diff
@@ -840,10 +845,10 @@ prefixNonemptyTextEncoder p = Encoder $ pure $ EncoderImpl
   }
 
 packTextEncoder :: (Applicative check, Applicative parse, IsText text) => Encoder check parse String text
-packTextEncoder = isoEncoder packed
+packTextEncoder = viewEncoder packed
 
 unpackTextEncoder :: (Applicative check, Applicative parse, IsText text) => Encoder check parse text String
-unpackTextEncoder = isoEncoder unpacked
+unpackTextEncoder = viewEncoder unpacked
```

Comment:
Didn't I already do this in https://github.com/obsidiansystems/obelisk/pull/1075?

Reply:
Yes, this PR is built with parts of https://github.com/obsidiansystems/obelisk/pull/1075 . I thought I cherry-picked the commits or at least marked you as a co-author, but apparently the commit history disagrees... You can force push necessary changes if you want or build a new PR using parts of this one if you want.

Reply:
It's fine either way, I just don't understand how github produces this diff when both develop and the PR branch have `viewEncoder`

Reply:
I think the diff is stuck at the old `develop` due to merge conflicts. Merging `develop` into this branch will probably resolve the issue.